Jenkinsfile_CNP

```diff
@@ -99,9 +93,6 @@ withPipeline(type, product, component) {
 
   afterAlways('smoketest:preview') {
     steps.archiveArtifacts allowEmptyArchive: true, artifacts: 'test-results/smokeTest/**/*'
-    if (checkForEnableHelmLabel(env.BRANCH_NAME)) {
```

Comment:
why are we deleting this ? As we still have to interrupt the build incase of helm enabled

Reply:
I agree with you.
I back with this part of the code.